googler

```diff
@@ -1175,13 +1206,23 @@ class GoogleParser(html.parser.HTMLParser):
             self.set_handlers_to('result')
             return 'result'
 
+        # Autocorrect
+        if tag == 'span' and 'spell_orig' in self.classes(attrs) and attrs.get('id') != 'sifm':
+            self.autocorrected = True
+            return
+        if tag == 'a' and 'spell' in self.classes(attrs) and attrs.get('id') != 'srfl':
```

Comment:
Doesn't this condition alone indicate auto-correction? I am asking because we already know `spell_orig`... we are the ones passing the search keywords. If possible, I'ld love to skip searching for it.

Also, the reason I am behaving like a miser here is: we do around 7000 pattern matches and 800+ tag parsing for a page on an average.